pkg/mutation/path/parser/parser.go

```diff
@@ -22,6 +22,12 @@ import (
 	"github.com/open-policy-agent/gatekeeper/pkg/mutation/path/token"
 )
 
+// Base errors for
+var (
```

Comment:
Can we use [Dave Cheney's trick](https://dave.cheney.net/2016/04/07/constant-errors) to define these as const?

Reply:
It [isn't idiomatic](https://blog.golang.org/go1.13-errors) - the blog post Dave Cheney wrote was written before Go 1.13. Let me know if using Dave Cheney's trick is a change you would like me to make to this PR.

Reply:
It seems to be completely compatible with Go 1.13 error wrapping and solves the problem of mutable sentinel values. That said, it's a suggestion, not a blocker - so up to you.